mypy_json_report.py

```diff
@@ -29,12 +71,12 @@ class MypyError:
     message: str
 
 
-def produce_errors_report(input_lines: Iterator[str]) -> str:
+def parse_errors_report(input_lines: Iterator[str]) -> Dict[str, Dict[str, int]]:
     """Given lines from mypy's output, return a JSON summary of error frequencies by file."""
```

Comment:
This doc string is no longer accurate as the function doesn't return JSON any more.